### ISSUE TITLE
Customizable for umbrella review

### ISSUE THREAD

**Issue by shainatrevino** [prisma-flowdiagram/PRISMA2020#49]
I have a suggested feature request. I haven't been able to figure out how to add a new line in a box to report multiple numbers. I mostly conduct umbrella reviews, so in our PRISMA chart we often report many different numbers in a single flow chart box. For example, we often report the number of references (n) and reviews (k) assessed for eligibility. We aslo end up reporting the number of included reviews (k) and primary studies (j). It would be great to have a solution that allowed the user to choose the letter for the value reported (e.g., n, k, or j), and be able to report multiple in one box, if desired. One alternative could be to allow the user to modify the `n` variable in the template file with the full text of what they want shown ("n = 10; k = 5), rather than just numbers (10). Another option would be to allow the user to add boxes within existing nodes that would add a line in each box to provide more information.

**Comment by chriscpritchard** [comment]
Can you send me an example flow diagram for this so I can visualise exactly what you want? This may be possible but might take some time!

**Comment by shainatrevino** [comment]
Thank you for your response! I have attached an example of a PRISMA diagram we created for an umbrella review. It doesn't have all the components from the PRISMA2020 diagram but does show the need to have multiple numbers/levels of reporting for based on whether you are reporting number of citations, reviews, or studies. 
[DPO_prisma_diagram.docx](https://github.com/prisma-flowdiagram/PRISMA2020/files/11095069/DPO_prisma_diagram.docx)
